week1/src/backend/routes/large-meals.js

```diff
@@ -0,0 +1,5 @@
+const mealsWithReviews = require("./meals.js");
+
+let largeMeals = mealsWithReviews.filter( meal => {
+    return meal.maxNumberOfGuests > 5});
```

Comment:
You can write this like this:
```suggestion
let largeMeals = mealsWithReviews.filter( meal => meal.maxNumberOfGuests > 5);
```